src/adapter/input/post_router.py

```
@@ -20,16 +20,14 @@ def get_post_application_service():
 @router.post(path='', 
             status_code=status.HTTP_201_CREATED,
             responses={
-                403: {"description": "인증 코드가 올바르지 않음"},
-                500: {"description": "서버 내부 오류"}
-            })
-def create_post(post: Post, service: PostApplicationService = Depends(get_post_application_service)):
+                403: {"description": "Authentication Code Error"},
+                500: {"description": "Internal Server Error"}
+            },
+            response_model=PostCreateResponse)
+def create_post(post_create_request: PostCreateRequest, service: PostApplicationService = Depends(get_post_application_service)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)